### PR TITLE
[minor] Fix wml_url and detect wml version

### PR DESCRIPTION
**Pull request by lokesh-sreedhara**
https://github.ibm.com/maximoappsuite/tracker-cicd/issues/613
https://github.ibm.com/maximo/Asset-Health-Insight/issues/12680

Fix broken predict fvt
- Predict activation fails when cpd_url is not available #613

The WML url can be a fixed Internal WML url in this case. WML version is not equal to Cp4D Version

=======

Update from @alequint :

- I did an additional commit to automate some of the configuration here: https://github.com/ibm-mas/ansible-devops/pull/509/commits/f4f3f7447cd59ed292db73fe7da6dd98e2b8634d + docs + some updates (@lokesh-sreedhara for your review)

- [x] Test evidence setting cpd_product_version as 4.0.9:

image

- [x] Test evidence setting cpd_product_version as 4.5:

image

- [x] Test evidence setting mas_app_settings_wml_version

image

- [x] Test in FVT Pipeline - @caroazad working on it

### CODE REVIEW

File: docs/playbooks/oneclick-predict.md
Comment (durera) [inline docs/playbooks/oneclick-predict.md:29]: Instead of requiring user to provide WML_VERSION and CPD_VERSION ... based on the above, can you add code to calculate wml_version automatically.

e.g. 

- if cpd_version starts with 4.0 wml_version = 4.0
- if cpd_version starts with 4.5 wml_version = 4.5

File: ibm/mas_devops/roles/suite_app_config/vars/defaultspecs/predict.yml
Comment (durera) [inline ibm/mas_devops/roles/suite_app_config/vars/defaultspecs/predict.yml:10]: The lookups should all live in the defaults/main.yml file of the role, and just reference them vars here.

**Review by durera — CHANGES_REQUESTED** [review]

File: docs/playbooks/oneclick-predict.md
Comment (lokesh-sreedhara) [reply docs/playbooks/oneclick-predict.md:29]: to unblock predict_fvt, can we get this merged as user provided?. i will create another PR and automate WML_VERSION later this week

File: ibm/mas_devops/roles/suite_app_config/vars/defaultspecs/predict.yml
Comment (lokesh-sreedhara) [reply ibm/mas_devops/roles/suite_app_config/vars/defaultspecs/predict.yml]: fixed, moved lookup to defaults/predict.yml

**Review by alequint — DISMISSED** [review]

File: docs/playbooks/oneclick-predict.md
Comment (alequint) [reply docs/playbooks/oneclick-predict.md]: @lokesh-sreedhara @durera  pls review my changes

**Comment by alequint** [comment]
Updating to be a draft PR. @lokesh-sreedhara before we merge this let's test predict installation, using fvt pipeline.

File: docs/playbooks/oneclick-predict.md
Comment (lokesh-sreedhara) [reply docs/playbooks/oneclick-predict.md]: @alequint thank you. the changes looks good to me. nice work!

**Comment by caroazad** [comment]
Configured vars in cli and ansible-fvt. Test results were positive. Image below
Screenshot 2022-11-16 at 10 43 43 PM

**Pull request merged**
